processing/src/main/java/org/apache/druid/query/lookup/LookupSegment.java

```diff
@@ -80,5 +79,12 @@ public Function<Map.Entry<String, String>, Object> columnFunction(String columnN
         },
         ROW_SIGNATURE
     );
+    this.lookupName = lookupName;
+  }
+
+  @Override
+  public String asString()
+  {
+    return "lookup." + lookupName;
```

Comment:
Nit: Since the default impl prints the class name, we could do
```suggestion
    return getClass().getSimpleName() + ":" + lookupName;
```